index.html

```diff
@@ -187,6 +187,22 @@ <h6>Skills</h6>
 
         </div>
     </section>
+    <section class="contact-section">
+       <div class="contact-container" >
+        <div class="contact-heading">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project
+                that needs coding. I’d love to help with it </p>
+        </div>
+        <form class="contact-form" action="https://formspree.io/f/mgedkazg" method="POST">
+            <input class="name" type="text" maxlength="30" name="name" placeholder="Name" required />
+            <input class="email" type="email" name="email" placeholder="Email Address" required />
+            <textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea>
+            <button class="submit-btn" type="submit">Get It Touch</button>
+        </form>
+       </div>
+
+    </section>
```

Comment:
- [x] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is missing the bottom line from the form. Please place a line at bottom of your form and make sure it matches the design.

Check this image for reference

![Screenshot (118)](https://user-images.githubusercontent.com/83803414/154142132-406ca590-76da-40d6-a20d-5558560798a6.png)

Reply:
I am convinced that this line represents the iPhone gesture mark and it is not part of the design . This is used on every mobile view at the bottom. @thecodechaser

Reply:
> I am convinced that this line represents the iPhone gesture mark and it is not part of the design . This is used on every mobile view at the bottom. @thecodechaser

@isaacpitwa Hello, Isaac. Thank you for reaching out, I know this line represents the iphone gesture mark but as CR We have some guidelines to follow and make sure the students meet those guidelines, Please add that line to your form.